open-craft/ecommerce
# OC-4077, OC-4078 and OC-4079 Restyle receipt page and reenroll error page

Restyle coupon redeem page - error message.

**Screenshots**: 
OC-4079: ![screenshot-2018-5-4 redeem](https://user-images.githubusercontent.com/1730454/39618306-6fd3f0c8-4f83-11e8-8f9f-8da183e44ca5.png)

OC-4077 / OC-4078: ![screenshot-2018-5-7 receipt for edx-100029 1](https://user-images.githubusercontent.com/1730454/39699095-f97af0ee-51f7-11e8-88b2-7f8a71bdea1d.png)

**Merge deadline**: None

**Testing instructions**:

For OC-4079:
1. create a coupon that can be redeemed just once
2. redeem it twice

For OC-4077 / OC-4078:
1. create a discount coupon with a 0 < discount < 100
2. redeem it and check the receipt page

**Reviewers**
